src/include/daos_prop.h

```diff
@@ -289,26 +289,35 @@ enum {
 	DAOS_PROP_CO_UNCLEAN,
 };
 
+/** clear the UNCLEAN status */
+#define DAOS_PROP_CO_CLEAR	(0x1)
 struct daos_co_status {
 	/* DAOS_PROP_CO_HEALTHY/DAOS_PROP_CO_UNCLEAN */
-	uint32_t	dcs_status;
+	uint16_t	dcs_status;
+	/* flags for DAOS internal usage, DAOS_PROP_CO_CLEAR */
+	uint16_t	dcs_flags;
 	/* pool map version when setting the dcs_status */
 	uint32_t	dcs_pm_ver;
 };
 
-#define DAOS_PROP_CO_STATUS_VAL(status, pm_ver)				\
-	((((uint64_t)(status)) << 32) | ((uint64_t)(pm_ver)))
+#define DAOS_PROP_CO_STATUS_VAL(status, flag, pm_ver)			\
+	((((uint64_t)(status)) << 48)		|			\
```

Comment:
Seems can simply change the bit sequence to be:
flags (highest 16bits) | status (47 ~ 32bits) | pm_ver (0~31bits).

1.2 client always set the highest 16bits as zero, that seems fine because it does not support the perm change.